[InfoBarInstantRecord] improvize code

Dima73 opened the pull request:
- add more information to warning messages output
- change the logic of displaying messages / time duration of records -code cosmetics
-add Hotkey option "Start recording current event" -set always 24 h when use Hotkey option "Start instant recording"

Taapat commented on lib/python/Screens/InfoBarGenerics.py:
```diff
@@ -2495,7 +2495,7 @@ def __init__(self):
 
 	def moveToTrash(self, entry):
-		print("instantRecord stop and delete recording: ", entry.name)
+		print("[instantRecord] stop and delete recording: %s", entry.name)
```
Is it really intended here print symbol %s?

Taapat commented on lib/python/Screens/InfoBarGenerics.py:
```diff
@@ -2562,66 +2562,87 @@ def getProgramInfoAndEvent(self, info, name):
+	def startInstantRecording(self, limitEvent=""):
 		begin = int(time())
-		end = begin + 3600      # dummy
+		end = begin + 3600 # 1h
```
If you are changing the comment format, why not change it according to PEP with two spaces from the statement?
https://peps.python.org/pep-0008/#inline-comments

Taapat commented on lib/python/Screens/InfoBarGenerics.py:
```diff
@@ -2562,66 +2562,87 @@ def getProgramInfoAndEvent(self, info, name):
 
 	def isInstantRecordRunning(self):
-		print("self.recording:", self.recording)
+		#print("[InfoBarInstantRecord] self.recording:", self.recording)
```
If you are commenting a line, why not do it according to PEP with a single space after the #?
https://peps.python.org/pep-0008/#inline-comments

Taapat commented:
I commented only on the first place where I think correction is needed to show what is wrong.
It would be good if you check all the proposed changes.
Print without keyword is also introduced in lines 2739 and 2748.
Two spaces from the statement are also missing in lines 2601, 2607 and 2612.

Dima73 committed:
[InfoBarInstantRecord] improvize code

- add more information to warning messages output
- change the logic of displaying messages / time duration of records
-code cosmetics
-add Hotkey option "Start recording current event"
-set always 24 h when use Hotkey option "Start instant recording"
